Library/Formula/pulseaudio.rb

```diff
@@ -1,21 +1,30 @@
-require 'formula'
-
 class Pulseaudio < Formula
   homepage "http://pulseaudio.org"
-  url "http://freedesktop.org/software/pulseaudio/releases/pulseaudio-5.0.tar.xz"
-  sha1 "e420931a0b9cf37331cd06e30ba415046317ab85"
+  url "http://www.freedesktop.org/software/pulseaudio/releases/pulseaudio-6.0.tar.xz"
+  mirror "https://mirrors.kernel.org/debian/pool/main/p/pulseaudio/pulseaudio_6.0.orig.tar.xz"
+  sha256 "b50640e0b80b1607600accfad2e45aabb79d379bf6354c9671efa2065477f6f6"
+
+  head do
+    url "http://anongit.freedesktop.org/git/pulseaudio/pulseaudio.git"
+    depends_on "automake" => :build
+    depends_on "autoconf" => :build
+  end
 
   option "with-nls", "Build with native language support"
   option :universal
 
   depends_on "pkg-config" => :build
   depends_on "libtool" => :build
-  depends_on "intltool" => :build if build.with? "nls"
-  depends_on "gettext" => :build if build.with? "nls"
+
+  if build.with?("nls") || build.head?
+    depends_on "intltool" => :build
+    depends_on "gettext" => :build
+  end
```

Comment:
Repeat these deps inside the `head` block rather than the `|| build.head?`. Going forward, any use of `build.head?`, `build.devel?`, or `build.stable?` in the class body (i.e. in the DSL section, not the install method) is to be avoided.